token/core/zkatdlog/crypto/issue/issue.go

```diff
@@ -14,59 +14,80 @@ import (
 	"github.com/hyperledger-labs/fabric-token-sdk/token/core/zkatdlog/crypto/rp"
 	"github.com/hyperledger-labs/fabric-token-sdk/token/core/zkatdlog/crypto/token"
 	"github.com/hyperledger-labs/fabric-token-sdk/token/driver"
+	token2 "github.com/hyperledger-labs/fabric-token-sdk/token/token"
 	"github.com/pkg/errors"
 )
 
-// IssueAction specifies an issue of one or more tokens
-type IssueAction struct {
+// Action specifies an issue of one or more tokens
+type Action struct {
 	// Issuer is the identity of issuer
-	Issuer []byte
-	// OutputTokens are the newly issued tokens
-	OutputTokens []*token.Token `protobuf:"bytes,1,rep,name=outputs,proto3" json:"outputs,omitempty"`
+	Issuer driver.Identity
+	// Inputs specify the identifiers in of the tokens to redeem
+	Inputs []*token2.ID
```

Comment:
From an interface perspective it would be maybe more clear to have `Inputs` as an aggregation of both id and token
```
Inputs []struct {
 ID *token2.ID
 Token []byte
}
```

Reply:
okay